azext_iot/tests/iothub/test_iot_hub_unit.py

```diff
@@ -0,0 +1,341 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
```

Comment:
This and a couple other test file re-additions shows this is not a safe merge. Compare with the state of the repository in `device_client_integration`, test files have been shifted around.

Reply:
Yeah, this needs to be added to the core directory